src/flamegraph.cpp

```diff
@@ -680,6 +799,23 @@ void FlameGraph::saveSvg(const QString& fileName) const
     m_rootItem->setBrush(oldBrush);
 }
 
+void FlameGraph::setColorScheme(ColorScheme _scheme)
+{
+    scheme = _scheme;
+
+    std::function<void(FrameGraphicsItem*)> traverseFunction = [&traverseFunction](FrameGraphicsItem* item) {
+        item->setBrush(brush(item->symbol(), BrushType::Hot));
```

Comment:
this is not setting the correct brush value

Reply:
what is the correct value?